build.sbt

```diff
@@ -55,7 +55,10 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     buildInfoSettings,
     libraryDependencies ++= Seq(
       metaconfig.value,
-      scalameta.value
+      scalameta.value,
+      // scala-reflect is an undeclared dependency of fansi, see #1252.
```

Comment:
I generally try to include the full link since it makes it easier to open from the editor.

Reply:
Aah, I see the issue link got shortened from my review comment, the original markup had the full link.

Reply:
It's fine, don't worry about it 😉